doc/installing/universal.md

```diff
@@ -158,21 +181,21 @@ $ python3 install.py user --version=3.10
 
 To uninstall the engine, use the --remove flag:
 
-on macOS and Linux:
-```
-$ python3 install.py devel --remove
-$ python3 install.py user --remove
-$ sudo -H /usr/bin/python3 install.py server --remove
-```
-
-on Windows:
+**on Windows:**
 ```
 C:\>cd %USERPROFILE%
 C:\>python install.py devel --remove
 C:\>python install.py user --remove
 ```
 
-The calculation data (in the `oqdata` directory) WILL NOT be removed.
+**on macOS and Linux:**
+```
+$ python3 install.py devel --remove
+$ python3 install.py user --remove
+$ sudo -H /usr/bin/python3 install.py server --remove
+```
```

Comment:
I understand that this part does not work on macOS, there is no /usr/bin/python3 .

Reply:
I will change this myself